lib/src/components/NxDropdown/NxDropdown.tsx

```diff
@@ -38,7 +38,10 @@ const NxDropdown: FunctionComponent<Props> = function NxDropdown(props) {
     <NxButton type="button"
               variant={variant || 'tertiary'}
               className={buttonClasses}
-              onClick={!disabled && onToggleCollapse || undefined}>
+              onClick={!disabled && onToggleCollapse || undefined}
+              aria-haspopup="true"
+              aria-pressed={isOpen ? true : false}
+              aria-expanded={isOpen ? true : false}>
```

Comment:
These seem redundant with each other to me.  ChromeVox reads the gallery example as "Navigation. Popup button expanded pressed" or "Navigation. Popup button collapsed not pressed".

Reply:
> These seem redundant with each other to me. ChromeVox reads the gallery example as "Navigation. Popup button expanded pressed" or "Navigation. Popup button collapsed not pressed".

I went back and re-read the MDN entry for pressed and I think you're right, it's for something that acts as a toggle control. Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/212/commits/96f2d4d67404f48081ee15b76ddc3e3ef0085913